proposals/js/JsName.md

```diff
@@ -81,6 +88,108 @@ Additionally it can be used to force using functions for accessors to avoid prob
     - *translates an accessing to such property as call to accessor function using name from the annotation.*
 
 
+## How it should work for overrides?
+
+Actually, the problems actual even we prohibit using `JsName` on overrides,
+because the compiler doesn't mangle the name of native declarations,
+so we can simply get name clash when to inherit from two natives.
+
+And doesn't make sense to prohibit to use this annotation on non-final declarations.
+
+So propose to **prohibit using JsName on overrides**, but **report about name clashes** on non-native declarations.
+
+
+**Case 1: declarations with different names from different parents with the same requested name (by JsName)**
+    _(map many names to the one)_
+
+
+_Note: A, B, C can be native._
```

Comment:
>What about none of A, B, C are native, but overridden method(s) is/are native.
I don't see what should be changed. Please provide a code fragment if you want to discuss it.

> What about fake override? 
see second example

> What about bar having implementation in one of (both of) A and B?

It already covered by current language rules